src/client/executor.rs

```diff
@@ -117,7 +117,6 @@ impl Client {
                     }
                     .into());
                 }
-
```

Comment:
Any particular reason for the despacing?  I find that putting blank lines between groups of related statements helps when visually skimming.